sql/driver/doc.go

```diff
@@ -235,6 +238,47 @@ Querying rows:
 
 	SELECT __key, name FROM person WHERE age > 30
 
+2) "json" is a first-class SQL type with IMap and Kafka Connector support.
+
+Creating a mapping:
+
+	CREATE MAPPING person (
+		__key BIGINT,
+		this  JSON
+	)
+	TYPE IMAP
+	OPTIONS (
+		'keyFormat' = 'bigint',
+		'valueFormat' = 'json'
+	)
+
+Inserting rows:
+
+	// Use serialization.JSON type to specify a JSON value.
+	INSERT INTO person VALUES(100, serialization.JSON(fmt.Sprintf(`{"age":%d, "name":%s}`, 35, 'Jane Doe')))
+
+Querying rows:
+Error handling is omitted for to keep the example short.
```

Comment:
"omitted to keep"

Reply:
Thanks fixed at https://github.com/hazelcast/hazelcast-go-client/pull/726/commits/425450077e2cfc16fa1a0d8d53b09fa85f8a7f11